src/biogeophys/WaterStateType.F90

```diff
@@ -346,7 +347,11 @@ subroutine InitCold(this, bounds, &
                   if (j > nbedrock) then
                      this%h2osoi_vol_col(c,j) = 0.0_r8
                   else
-                     this%h2osoi_vol_col(c,j) = 0.15_r8 * ratio
+                     if(use_fates_planthydro) then
+                         this%h2osoi_vol_col(c,j) = 0.75_r8*watsat_col(c,j)*ratio
```

Comment:
What is this 0.75 hardcoded constant? Can it be a parameter?

Reply:
@rgknox can you weigh in here?

Reply:
This will be addressed later.